ai_edge_torch/generative/utilities/stable_diffusion_loader.py

```diff
@@ -743,6 +743,10 @@ def load(
 
     attention_config = layers_config.AttentionConfig(
```

Comment:
attnetion_config should have the same update as the one in diffusion.py.
Though in the model loader, the `head_dim` is never used, so the variable here doesn't matter.
Either add a comment for reminding this is not the actual config used in the model, or update to per layer config here

Reply:
+1

Reply:
Thanks @yichunk. I have just now fixed this with an additional commit. @haozha111 if you can LGTM again I willl merge in these changes.